site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,463 @@
+
+To compare waterfalls, expand the **Waterfall Opacity** section and drag the slider. To view the video, click **Adjust Filmstrip Settings**, scroll down inside that dialog, and click **View Video**.
+
+If the SXG prefetch is successful, you will see that the "with SXG" waterfall doesn't include a row for the HTML, and the fetches for subresources start sooner. For example, compare "Before" and "After" here:
+
+<web-tabs>
+  <web-tab title=Before>
+    {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/LwbWyWbcX3oHnM8lSwDA.png", alt="Network waterfall without SXG prefetch; first row is HTML fetch which takes 1050ms", width="800", height="327" %}
+  </web-tab>
+  <web-tab title=After>
+    {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/JPmTMO0WYMJ7aY008L4V.png", alt="Network waterfall with SXG prefetch; the HTML has been prefetched, allowing all subresources to start fetching 1050ms earlier", width="800", height="327" %}
+  </web-tab>
+</web-tabs>
+
+## Debug
+
+If the WebPageTest is showing that the SXG is being prefetched, then it has succeeded in all the steps of the pipeline; you may skip to the [Optimize](#optimize) section to learn how to further improve LCP. Otherwise, you'll need to find out where in the pipeline it failed and why; read on to learn how.
+
+### Publishing
+
+Make sure your pages are being generated as SXGs. To do so, you need to pretend to be a crawler. The easiest way is to use the [SXG Validator Chrome extension](https://chrome.google.com/webstore/detail/sxg-validator/hiijcdgcphjeljafieaejfhodfbpmgoe):
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/zUqSkRSLNkF26gjyymEo.png", alt="SXG Validator showing a check mark (✅) and a Content Type of application/signed-exchange;v=b3", width="600", height="281" %}
+</figure>
+
+{% Aside %}
+SXGs are meant for SXG-capable crawlers, to enable prefetching. When users visit your site directly, you should not serve SXGs to them, since they will not be prefetched—doing so only adds overhead. Cloudflare ASX performs this [content negotiation](https://github.com/google/webpackager/blob/main/cmd/webpkgserver/README.md#content-negotiation) automatically.
+{% endAside %}
+
+The extension fetches the current URL with an `Accept` request header that says it prefers the SXG version. If you see a check mark (✅) next to Origin, that means an SXG was returned; you can skip to the [Indexing](#indexing) section.
+
+If you see a cross mark (❌), that means an SXG wasn't returned:
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/4T3la9HtoRDz8SK7lWGi.png", alt="SXG Validator showing a cross mark (❌) and a Content Type of text/html", width="483", height="162" %}
+</figure>
+
+If Cloudflare ASX is enabled, then the most likely reason for a cross mark (❌) is because a cache control response header prevents it. ASX looks at headers with the following names:
```

Comment:
```suggestion
If Cloudflare ASX is enabled, then the most likely reason for a cross mark (❌) is because a cache control response header prevents it. ASX looks at headers with the following names:

```